libopenage/renderer/camera/definitions.h

```diff
@@ -58,10 +60,18 @@ static constexpr float DEFAULT_MAX_ZOOM_OUT = 64.0f;
  */
 static constexpr float DEFAULT_ZOOM_RATIO = 1.0f / 49;
 
+static constexpr CameraBoundaries DEFAULT_CAM_BOUNDARIES{
+	std::numeric_limits<float>::lowest(),
+	std::numeric_limits<float>::max(),
+	std::numeric_limits<float>::lowest(),
+	std::numeric_limits<float>::max(),
+	std::numeric_limits<float>::lowest(),
+	std::numeric_limits<float>::max()};
+
 /**
  * Constant values for the camera bounds.
  * TODO: Make boundaries dynamic based on map size.
  */
-static const float X_MIN = 12.25f, X_MAX = 32.25f, Z_MIN = -8.25f, Z_MAX = 12.25f;
+static constexpr float X_MIN = 12.25f, X_MAX = 32.25f, Z_MIN = -8.25f, Z_MAX = 12.25f;
```

Comment:
I guess we should also define bounds for the Y axis. They should probably be

- min: `0.0f`
- max: `20.0f`